magento/magento2
# [Catalog|MediaStorage] Use theme for image frame param calculation

### Description (*)

This PR replace for https://github.com/magento/magento2/pull/33413 with fixed unit tets
Full description available in issue magento/magento2#19710

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#19710

### Manual testing scenarios (*)


1. Assign images to products in the catalog.
2. Set the value of the variable "product_image_white_borders" to 0 in your custom theme view.xml file.
3. Regenerate product images cache with the CLI command catalog:images:resize.
4. The product thumbnail in category pages should appear

### Questions or comments


### Contribution checklist (*)
